libs/common/async.go

```diff
@@ -118,7 +119,7 @@ func (trs *TaskResultSet) FirstError() error {
 // Parallel
 
 // Run tasks in parallel, with ability to abort early.
-// Returns ok=false iff any of the tasks returned abort=true.
+// Returns ok=false if any of the tasks returned abort=true.
```

Comment:
```suggestion
// Returns ok=false iff any of the tasks returned abort=true.
```

iff - if and only if ... is correct